src/averageAges.js

```diff
@@ -1,5 +1,13 @@
 'use strict';
 
+function avarageAge(people) {
+  const reducerPeople = people.reduce((prev, current) => prev
+  + (current.died - current.born), 0)
+   / people.length;
```

Comment:
```suggestion
  return people.reduce((prev, current) => prev
    + (current.died - current.born), 0) / people.length;
```

Reply:
Hey, you earlier  said  i can use variables, i did it, but now you write to delete it

Reply:
If u create var, and in the next line return one... it is not required :)